subsys/bluetooth/scan.c

```diff
@@ -791,6 +814,104 @@ static int scan_appearance_filter_add(u16_t appearance)
 	return 0;
 }
 
+static bool adv_manufacturer_data_cmp(const u8_t *data,
+		u8_t data_len,
```

Comment:
Alignment

Reply:
What's actually the correct alignment in this case?

Reply:
`function(aaaaaa, ...,
               bbbbbb, ..,
               ....)
`
Justify the arguments in the new lines to the first.